cxx/util_distribution_variant.cc

```diff
@@ -99,3 +103,18 @@ DistributionVariant cluster_prior_from_spec(
       assert(false && "Unsupported distribution enum value.");
   }
 }
+
+EmissionVariant cluster_prior_from_spec(const EmissionSpec& spec,
```

Comment:
This duplicates logic from get_emission.  I would prefer if we kept that as the single point of truth.

(Update:  I've addressed this in https://github.com/probcomp/hierarchical-irm/pull/81).